templates/zerver/emails/followup_day1.source.html

```diff
@@ -53,7 +53,10 @@
 </p>
 
 <p>
-    {% trans support_email=macros.email_tag(support_email) %}Questions? Contact
-us any time at {{ support_email }}.{% endtrans %}
+    {% if corporate_enabled %}
+        {% trans %}Questions? <a href="{{ support_email }}" >Contact us</a> — we'd love to help!{% endtrans %}
```

Comment:
I think for all the translation tags, we should add back `% trans support_email=macros.email_tag(support_email) %`. From my understanding, the translation tags converts the` support_mail` to `mailto:support_mail@zulip.com` using the following macro: 

`{% macro email_tag(email, text) -%}
    <a href="mailto:{{ email }}">{{ text | default(email) }}</a>
{%- endmacro %}`

This also replaces `<a href="{{ support_email }}" >Contact us</a>` with something more readable like `{{ support_email }}` instead by using the macro.

@laurynmm thoughts?

Reply:
But we do need the "Contact us" text in the HTML anchor tag for Zulip Cloud to be translated. I don't see any examples currently in the emails where the text parameter is passed for the `email_tag` macro, so I wonder if it would be visible as part of the string needing to be translated if it was used.

Here are the strings that are generated by these new macros currently:
```po
#: templates/zerver/emails/macros.source.html:14
#, python-format
msgid ""
"If you have any questions, please contact this Zulip server's administrators "
"at <a href=\"%(email)s\">%(email)s</a>."
msgstr ""

#: templates/zerver/emails/macros.source.html:18
#, python-format
msgid ""
"Do you have questions or feedback to share? <a href=\"%(email)s\">Contact "
"us</a> — we'd love to help!"
msgstr ""
```

This seems similar to other types of links, for example in `missed_message.source.html`...
```po
#: templates/zerver/emails/missed_message.source.html:17
#, python-format
msgid ""
"This email does not include message content because your organization has "
"disabled <a class=\"content_disabled_help_link\" href=\"%(help_url)s"
"\">message content appearing in email notifications</a>."
msgstr ""

#: templates/zerver/emails/missed_message.source.html:19
#, python-format
msgid ""
"This email does not include message content because you have disabled <a "
"class=\"content_disabled_help_link\" href=\"%(alert_notif_url)s\">message "
"content appearing in email notifications</a>."
msgstr ""
```